test/conformance/queue/urQueueGetInfo.cpp

```diff
@@ -55,12 +55,30 @@ TEST_P(urQueueGetInfoTestWithInfoParam, Success) {
             ASSERT_EQ(*returned_device, device);
             break;
         }
+        case UR_QUEUE_INFO_DEVICE_DEFAULT: {
+            auto returned_default_queue =
+                reinterpret_cast<ur_queue_handle_t *>(data.data());
+            ASSERT_EQ(*returned_default_queue, queue);
+            break;
+        }
+        case UR_QUEUE_INFO_FLAGS: {
+            auto returned_flags =
+                reinterpret_cast<ur_queue_flags_t *>(data.data());
+            EXPECT_EQ(*returned_flags, *returned_flags & UR_QUEUE_FLAGS_MASK);
+            break;
+        }
         case UR_QUEUE_INFO_REFERENCE_COUNT: {
             auto returned_reference_count =
                 reinterpret_cast<uint32_t *>(data.data());
             ASSERT_GT(*returned_reference_count, 0U);
             break;
         }
+        case UR_QUEUE_INFO_EMPTY: {
+            auto returned_empty_queue =
+                reinterpret_cast<ur_bool_t *>(data.data());
+            ASSERT_TRUE(returned_empty_queue);
+            break;
+        }
         default:
             break;
```

Comment:
If we intend to check every info type (or at the very least, call attention every info type), should we not return an error here rather than silently doing nothing?